include/mpsl/mpsl_work.rst

```diff
@@ -0,0 +1,21 @@
+.. _mpsl_work:
+
+Multiprotocol Service Layer workqueue
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Multiprotocol Service Layer workqueue library makes it possible submit work to be processed by the :ref:`nrfxlib:mpsl` workqueue.
+
+The Multiprotocol Service Layer workqueue is a workqueue running with a higher priority than the system workqueue.
+
```

Comment:
Could you extend the documentation adding also the following items?
- an implementation/architecture description
- any requirements needed for using it
- location of the library files
- list of samples using the library (if any is present in ncs)
- any additional information that might be needed, like for example how to use it in a custom application
- dependencies

Reply:
I've updated the documentation. I hope this is satisfactory. There is not much documentation to add as this is simply an instantation of a workqueue, similar to the system workqueue. It now has more documentation than implementation.